clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -202,6 +203,8 @@ public void releaseCache(String name) throws ClusteredTierReleaseException, Time
   public void destroyCache(String name) throws ClusteredTierDestructionException, TimeoutException {
     try {
       invokeInternal(timeouts.getLifecycleOperationTimeout(), messageFactory.destroyServerStore(name), true);
+    } catch (ResourceBusyException e) {
```

Comment:
Did not think of going that far. Anyway, what gets out of Ehcache is a `CachePersistenceException` so in the end,  does not matter too much. Was mostly saying that I would handle _valid_ outcomes specifically rather than in the general catch block below.